_includes/schedule.md

```diff
@@ -37,11 +37,17 @@
                 }
             },
             "courseMode": ["online", "synchronous"],
+            "courseWorkload": "90-minutes long cohort call",
             "startDate" : "{{ c.date | date: "%Y-%m-%d" }}",
             "endDate" :"{{ c.date | date: "%Y-%m-%d" }}",
             "inLanguage": "en-US",
             "duration": "{{ c.duration }}",
-            "name" : "{{ c.title }}"
+            "name" : "{{ c.title }}",
+            "offers": {
```

Comment:
I think it should be a list of `offer` element (here just 1)

Reply:
I had thought that since this is written within a loop, 1 would suffice. The way it's implemented now, it shows the price for every instance of `calls`.

Can you please clarify what you have in mind?

Reply:
I'll give it a run locally, and show you the output.

That might take roughly 20mins because of Mermaid. 😅

Reply:
So, it does work...pretty much exactly how it did before. The difference is that we now have a list of `offers`. 

That brings me to my question - if we want to specify a list, I think we have to reach out to Patri for accurate prices, no?

![schema_PR2](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/d8941976-8fcd-46b1-97e6-28f344fb25f5)

I also foresee a time when we regularly have to manually update the schema as new events come up with different prices.

Reply:
Here all calls in a cohort were for free. I would not bother too much for now

Reply:
True. 💯